cloud-pack/cloud-pack.config/src/main/content/jcr_root/apps/streamx-connector-aem-cloud-pack/osgiconfig/config.author/dev.streamx.sling.connector.paths.PathsExtraction~pages.cfg.json

```diff
@@ -0,0 +1,10 @@
+{
+  "search.regexes": [
+    "(/[^\"'\\s]*\\.coreimg\\.[^\"'\\s]*)",
+    "(/[^\"'\\s]*etc\\.clientlibs[^\"'\\s]*)"
```

Comment:
if clientlibs are here then add also handler for clientlibs. If you plan to add it  in next PRs then remove it from here